news-app/src/App.js

```diff
@@ -1,17 +1,20 @@
 import './App.css';
 import './header.css';
 import './topic-gallery.css';
+import './singletopicarticles.css';
```

Comment:
These css files are starting to add up a bit, I would maybe get them in a folder to organise them 👍